tests/multipleBackend/backendHealthcheckResponse.js

```diff
@@ -16,14 +14,4 @@ describe('Healthcheck response', () => {
             done();
         });
     });
-    it('should return result for every location constraint in ' +
```

Comment:
we shouldn't just delete this test.  we should test to make sure at least one of every azure and aws backend is getting called.